src/ia-dropdown.ts

```diff
@@ -109,11 +166,19 @@ export class IaDropdown extends LitElement {
   render() {
     return html`
       <div class="ia-dropdown-group">
-        <button @click=${this.toggleOptions} class="click-main">
+        <button @click=${this.mainButtonClicked} class="click-main">
           <span class="cta sr-only">Toggle ${this.optionGroup}</span>
           <slot name="dropdown-label"></slot>
           ${this.displayCaret
-            ? html`<span class="caret">${this.caret}</span>`
+            ? html`<span
+                class="caret"
+                tabindex=${this.openViaCaret && !this.openViaButton
```

Comment:
q: do you think we should have tabindex for caret always?

Reply:
Good question!

My feeling on this was, if the main button itself is a valid way to open the dropdown, then making the caret tabbable too would just add a redundant extra keypress. So I only want the caret to accept tab focus if the button itself is set _not_ to open the dropdown. Open to alternative approaches though!